### ISSUE TITLE
permission.request() callback never Called

### ISSUE THREAD

**Issue by DimDL** [delba/Permission#10]
On :head, permission.request() is broken, completion is never called.
With version 1.1, it works as expected (completion is called)

Test: Put this in you app delegate didFinishLaunching :

```
        let permission: Permission = .LocationWhenInUse
        print(permission.status) // => "NotDetermined"

        permission.request { status in
            switch status {
            case .Authorized:    print("authorized")
            case .Denied:        print("denied")
            case .Disabled:      print("disabled")
            case .NotDetermined: print("not determined")
            }
        }
```

Only .NotDetermined is printed

**Comment by delba** [comment]
I'm on my phone and can't give it a test now but after peeking into `requestLocationAlways|WhenInUse`, there is definitely something wrong indeed 😨

I'll fix it later in the day (hopefully) and release a new version short after.

But just for info, you did add a NSLocationWhenInUseDescription in your .plist, right?

**Comment by DimDL** [comment]
I did. I also tested with .Contacts, same failure.

**Comment by delba** [comment]
Ok I found the bug :)

In internal func callbacks(), I call `self.callbacks` instead of `self.callback`

Could you please fix this typo and fill a PR please (I'm still on my phone).

Thanks

**Comment by delba** [comment]
Fixed in 9cb2581ff88ab735bfe7a81181395c9d76ffe4a6

Thanks for the issue!

**Comment by DimDL** [comment]
I saw you just fixed it. Awesome, thanks !
